### ISSUE TITLE
Accessibility: decorative graphicImage

### ISSUE THREAD

**Issue by hamsterbacke81** [primefaces/primefaces#13790]
For accessibility, it is sometime necessary to define a image as decorative by using an empty **alt** attribute. i.e. decorative element which should be skipped by screen reader.
Right now (tested with primefaces 15.0.0), when use graphicImage Component and pass a empty string as alt, such as 

`
<p:graphicImage value="/resources/Atom_Icon_Info.svg" alt=""/>
`

the alt-attribute would be ignored completely and it is rendered to

`
<img id="j_idt8" src="/resources/Atom_Icon_Info.svg?pfdrid_c=true">
`

Is there a way to keep the empty alt attribute?

**Comment by melloware** [comment]
Yep i bet it's being skipped because it is empty but you should be able to use JSF Passthrough Attributes and do...

```xml
<p:graphicImage value="/resources/Atom_Icon_Info.svg" pt:alt=""/>
```

or use `aria-hidden`

```xml
<p:graphicImage value="/resources/Atom_Icon_Info.svg" pt:aria-hidden="true"/>
```

**Comment by hamsterbacke81** [comment]
thanks, exactly what I needed.

**Comment by melloware** [comment]
I will still see if I can fix this as it is a valid use case but that workaround should work for now!

**Comment by melloware** [comment]
OK it looks like the code was trying to do this but only checking NULL not for `""` as well.
